kibbeh/src/ui/UserAvatar/SingleUser.tsx

```diff
@@ -45,20 +45,16 @@ export const SingleUser: React.FC<AvatarProps> = ({
 }) => {
   return (
     <div
-      className={`relative inline-block ${className}`}
+      className={`relative inline-block${className ? ` ${className}` : ""}`}
```

Comment:
I'll take the trailing space over ternary

Reply:
Done